extensions/ql-vscode/src/remote-queries/variant-analysis-monitor.ts

```diff
@@ -94,6 +89,46 @@ export class VariantAnalysisMonitor extends DisposableObject {
     return { status: 'CompletedSuccessfully', scannedReposDownloaded: scannedReposDownloaded };
   }
 
+  private scheduleForDownload(
+    scannedRepo: VariantAnalysisScannedRepository,
+    variantAnalysisSummary: VariantAnalysisApiResponse
+  ) {
+    void commands.executeCommand('codeQL.autoDownloadVariantAnalysisResult', scannedRepo, variantAnalysisSummary);
+  }
+
+  private shouldDownload(
+    scannedRepo: VariantAnalysisScannedRepository,
+    alreadyDownloaded: number[]
+  ): boolean {
+    return (!alreadyDownloaded.includes(scannedRepo.repository.id) && scannedRepo.analysis_status === 'succeeded');
```

Comment:
```suggestion
    return !alreadyDownloaded.includes(scannedRepo.repository.id) && scannedRepo.analysis_status === 'succeeded';
```
Unless I'm mistaken, the braces are unnecessary. And I'm not sure they add to readability.

Reply:
Removed: https://github.com/github/vscode-codeql/pull/1599/commits/7d2bae1f1b61c9234d5839701205d8e41797c9ba